### PR TITLE
fix: numGroups calculation

### PR DESCRIPTION
**Pull request by mrclickbits**
s. https://github.com/jquense/react-big-calendar/issues/1685

### CODE REVIEW

**Review by pietrofxq — APPROVED** [review]

File: src/utils/TimeSlots.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export function getSlotMetrics({ min: start, max: end, step, timeslots }) {
     1 + dates.diff(start, end, 'minutes') + getDstOffset(start, end)
```
Comment (pietrofxq) [inline src/utils/TimeSlots.js:19]: Wouldn't it be better to handle the +1 here? Won't removing it cause any problems?

Comment (mrclickbits) [reply src/utils/TimeSlots.js:19]: Although I'm not completely certain, I think the `totalMin` value is used for positioning items in the calendar as well while the `numGroups` is only used for figuring out the amount of slot groups. So the impact of changing the `numGroups` seems less risky to me.
Plus, my PR is inspired by the changes suggested in https://github.com/jquense/react-big-calendar/issues/1685 and an internal patch a colleague applied for fixing this behavior. And they both have the `- 1 ` expression in the `numGroups` value and its effect is working for us quite nicely.

**Comment by jhnns** [comment]
@jquense What are your thoughts on this change? 😀

**Comment by jquense** [comment]
i don't have any! happy to merge tho since someone approved it

**Pull request merged**